docs/2_2_common_mechanisms.adoc

```diff
@@ -661,8 +661,7 @@ The importer calls <<fmi3GetClock>> to inquire the clock activation state.
 |`countdown`
 |[[countdown,`countdown`]]<<fmi3SetClock>> in <<EventMode>>,
 <<fmi3ActivateModelPartition>> in <<ClockActivationMode>>,
-<<fmi3GetInterval>> in <<InitializationMode>> and in every <<EventMode>>,
-<<fmi3GetInterval>> in <<ClockActivationMode>> after every call to <<fmi3ActivateModelPartition>>
+<<fmi3GetInterval>> in <<InitializationMode>>, in every <<EventMode>>, and in <<IntermediateUpdateMode>>
```

Comment:
Can you add here : In Scheduled Execution the FMU has to inform the importer by calling fmi3CallbackIntermediateUpdate that the clock is about to tick.